src/items.ts

```diff
@@ -218,7 +218,9 @@ export function containerElement(
 /**
  * Item list (TODO: complete)
  */
-export const ITEMS = {
+export const ITEMS: {
+    [key: string]: any
+} = {
```

Comment:
Why is this change necessary? The inferred type is stronger then the declared type here.

Reply:
You are correct! Thanks for the fix. Implemented in https://github.com/lgfrbcsgo/du-factory-generator/commit/2660050714c6a7c71b8c8f3d3fa6d292a40db9eb